lib/sinatra/base.rb

```diff
@@ -348,7 +348,7 @@ def attachment(filename = nil, disposition = :attachment)
       response['Content-Disposition'] = disposition.to_s
       if filename
         params = '; filename="%s"' % File.basename(filename)
-        response['Content-Disposition'] << params
+        response['Content-Disposition'] += params
```

Comment:
@zzak I just ran across this in one of our projects. See https://github.com/sinatra/sinatra/pull/1137. We tried to freeze the constant we were using to set dispositions and got lots of errors..

Also this fixed a possible unintended side effect bug. If you hold a reference to the string you pass as the disposition, you'll find that string changed after the call.

```
x = 'inline'
send_file(some_path, disposition: x)
x == 'inline' #=> false
```